ManatiCR/FDI
# Feature/fdi 26 recursos juridicos content type

[FDI-26](https://manati.atlassian.net/browse/FDI-26)
-
**Steps to test**
- Run `ahoy drush cim -y`
- Log in as an admministrador de contenido user
- Create one node recurso jurídico, check you can edit it and delete it as well
